org.scala-ide.sdt.core.tests/src/org/scalaide/core/classpath/ClasspathTests.scala

```diff
@@ -240,7 +297,8 @@ class ClasspathTests {
       expectedWarnings = 0, expectedErrors = 2)
   }
 
-  /** Major binary-incompatible library on the classpath, with and without version suffix of their own.
+  /**
```

Comment:
We did in some other PR. The result was that we want to use

```
/**
 * Text here
 */
```

for multi line Scaladoc and `/** Text here */` for single line Scaladoc. There is just not yet a style guide somewhere that mentions this.